x-pack/plugins/ml/public/embeddables/single_metric_viewer/single_metric_viewer_embeddable.tsx

```diff
@@ -45,12 +46,56 @@ export class SingleMetricViewerEmbeddable extends Embeddable<
   private reload$ = new Subject<void>();
   public readonly type: string = ANOMALY_SINGLE_METRIC_VIEWER_EMBEDDABLE_TYPE;
 
+  // API
+  public readonly functionDescription: BehaviorSubject<string | undefined>;
+  public readonly jobId: BehaviorSubject<JobId | undefined>;
+  public readonly selectedDetectorIndex: BehaviorSubject<number | undefined>;
+  public readonly selectedEntities: BehaviorSubject<MlEntityField[] | undefined>;
+  public readonly title: BehaviorSubject<string | undefined>;
+
+  private apiSubscriptions = new Subscription();
+
   constructor(
     initialInput: SingleMetricViewerEmbeddableInput,
     public services: [CoreStart, MlDependencies, SingleMetricViewerServices],
     parent?: IContainer
   ) {
     super(initialInput, {} as AnomalyChartsEmbeddableOutput, parent);
+
+    this.jobId = embeddableInputToSubject<JobId, SingleMetricViewerEmbeddableInput>(
+      this.apiSubscriptions,
+      this,
+      'jobId'
```

Comment:
With the already released embeddable version in 8.13, you need to provide backward compatibility. 
The input prop used to be named `jobIds` and contained an array of strings instead of a string.

Reply:
This PR has a change in `single_metric_viewer_setup_flyout.tsx` which [resolves the array to a string](https://github.com/elastic/kibana/pull/179364/files#diff-18804fabd5258417f500214c7b87d059771ae3e27c16fe4866cc6f523d058f76R55) when setting up the panel `jobId: jobIds[0],`.

Reply:
Good call! Going to keep the original API for now to prevent any issues c39b883e6b82023461799d2c4ab78623359a1011